spec/models/api/base_component_spec.rb

```diff
@@ -118,6 +118,29 @@
     end
   end
 
+  describe '#sucrose?' do
+    subject(:component) do
+      described_class.new(
+        'measurement_unit_code' => measurement_unit_code,
+        'measurement_unit_qualifier_code' => measurement_unit_qualifier_code,
```

Comment:
does not work with symbols:
`measurement_unit_code: measurement_unit_code,
...
`

Reply:
Discussed. We can but it would be a PITA/require updating all specs to be consistent